.github/workflows/push-docker-image.yaml

```diff
@@ -2,17 +2,15 @@ name: publish docker image
 
 on:
   pull_request:
+    paths:
```

Comment:
Does it mean push will only trigger when these paths are modified?

Reply:
Only changes to this paths will trigger build check on pull request. We never push on pull request.